core/src/navigation/LeftNav.html

```diff
@@ -48,7 +48,7 @@
           <li class="fd-nested-list__item">
             <a
               href="{getRouteLink(node)}"
-              title="{$getTranslation(node.label)}"
+              title="{node.tooltipText ? $getTranslation(node.label) : ''}"
```

Comment:
What is the general idea behind the new `tooltipText` we are adding , or how is it different from the `label`. 

Unless something else was planned, it would be nice to maybe have this added in our documentation section: [navigation-parameters section](https://docs.luigi-project.io/docs/navigation-parameters-reference?section=navigation-parameters).